README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
+
+  -  create new .env file
+      add the following informations in it
+          SECRET_KEY
+          DATABASE_URL
+          DEBUG
+
+      NB for Secret key generate random variable of length 60
+      ```
+       import secrets
+       import string
+       ''.join(secrets.choice(string.printable) for i in range(60))
+       # random string will be generated and used as secret key for the app
+      ```
+
+There are some options to deploy apps to servers/fly.io. from those options docker is the one and good option to deploy with proper environment and it's easy to update and scalable
+- Doeckerization
+  * To dockerize the app the following 2 files will be created on the root of the project
```

Comment:
This is already done so why are you telling people to do it

Reply:
ok, i removed it